src/Flex/Flex.types.js

```diff
@@ -0,0 +1,41 @@
+export const FLEX_PROPS = {
+  alignItems: {
+    stretch: 'stretch',
+    center: 'center',
+    flexStart: 'flex-start',
+    flexEnd: 'flex-end',
+    baseline: 'baseline',
+    initial: 'initial',
+    inherit: 'inherit',
+  },
+  alignSelf: {
+    stretch: 'stretch',
+    center: 'center',
+    start: 'start',
+    end: 'end',
+  },
+  direction: {
+    column: 'column',
+    row: 'row',
```

Comment:
There are a couple other props here we should probably add:
```suggestion
    columnReverse: 'column-reverse',
    row: 'row',
    rowReverse: 'row-reverse',
```